iphone/Maps/Bookmarks/BookmarksList/BookmarksListViewController.swift

```diff
@@ -224,7 +224,11 @@ extension BookmarksListViewController: IBookmarksListView {
       actionSheet.addAction(action)
     }
     actionSheet.addAction(UIAlertAction(title: L("cancel"), style: .cancel, handler: nil))
-    actionSheet.popoverPresentationController?.barButtonItem = sortToolbarItem
+    let barButtonItem = switch source {
+    case .sort: sortToolbarItem
+    case .more: moreToolbarItem
```

Comment:
nit: Shouldn't case be indented?
```suggestion
      case .sort: sortToolbarItem
      case .more: moreToolbarItem
```

Reply:
I dont think so? at least based on this it says it should be at same level as switch statement

https://google.github.io/swift/#:~:text=Case%20statements%20are%20indented%20at,2%20spaces%20from%20that%20level.

Reply:
It's not a simple switch, it's a switch inside a let/if condition.